internal/filer_test.go

```diff
@@ -515,11 +564,14 @@ func TestAccFilerWorkspaceFilesExtensionsReadDir(t *testing.T) {
 		{"foo.r", "print('foo')"},
 		{"foo.scala", "println('foo')"},
 		{"foo.sql", "SELECT 'foo'"},
-		{"jupyterNb.ipynb", jupyterNotebookContent1},
-		{"jupyterNb2.ipynb", jupyterNotebookContent2},
+		{"py1.ipynb", readFile(t, "testdata/notebooks/py1.ipynb")},
+		{"py2.ipynb", readFile(t, "testdata/notebooks/py2.ipynb")},
```

Comment:
Why both for Python and not for the others?

Reply:
I don't recall the original reason why I added two notebooks here, but it does seem unnecessary. I have removed the second Python notebook from this test.